src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
+  constructor(name) {
+    this.name = name;
+  }
+  static init(obj) {
+    this.alive.push(obj);
+  }
 }
-
 class Herbivore extends Animal {
-  // write your code here
-}
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+
+    Herbivore.init(this);
```

Comment:
in order not to do the same action like  `Herbivore.init(this);` in Herbivore and `Carnivore.init(this);` in Carnivore
you can just write it one time in Animal class in constructor:  `Animal.init(this);`